### ISSUE TITLE
Accepted post

### ISSUE THREAD

**Issue by Riari** [Riari/laravel-forum#340]
The package should support an optional 'accepted post' feature, whereby a post can be marked as accepted by a user who has permission (by default, the thread author) to do so. This can be useful for Q&A categories, but could have other use cases.

**Comment by mcaccount83** [comment]
Does this feature have an expected roll out date?

**Comment by Riari** [comment]
I have no particular date in mind, it's just something I want to get around to eventually. Nobody has mentioned it before so I've not considered it a priority.

If it's something you're particularly interested in, I could try to focus on it over the next few weeks.

**Comment by mcaccount83** [comment]
I have a forum that I've been forced to keep in Google Groups, but I'd love to move to my laravel site so I don't have to maintain two sets of member lists.  But, my main hold up is that I need to have all posts be moderated/approved by a specific user and haven't been able to find a way to do that yet in laravel.  I've used your package with an internal group and it works great for what we need, just waiting on the approved post feature to use it on a large scale.  :)  Thanks for considering!

**Comment by Riari** [comment]
Ahh I see. Well, the feature you're talking about is a bit different from what I have in mind here. The "accepted post" feature would provide a similar mechanism to the "accepted answer" feature of Stack Overflow, but a mod approval feature also makes total sense.

Let me think on this a bit and see if I can do something to suit your needs. I'm away from home at the moment but will try to update you on Monday.

**Comment by Riari** [comment]
@mcaccount83 I've written up a new issue describing the feature here: #381. Feel free to reply to that issue if you have any feedback or questions. No promises on when it'll be delivered, but I plan to start working on it this week.
